tests/end2end/test_single_operator_with_cntk_backend.py

```diff
@@ -172,6 +173,12 @@ def test_dense(self):
 
         self.assertTrue(np.allclose(y_reference, y_produced))
 
+        # Verify Keras-to-ONNX path
+        onnx_model = onnxmltools.convert_keras(keras_model)
+        y_produced = _evaluate(onnx_model, x).reshape(N, D)
```

Comment:
Please add a note to explain the usage of reshape(N, D). E.g., the output shape, [1, N, D], from CNTK got an extra 1-element dimension, so we squeeze it with reshape. Please also copy this note to other places where Reshape is used.